### DISCUSSION TITLE
Is logic separated from view in DBeaver?

### DISCUSSION THREAD

**Discussion by PavelTurk** [dbeaver/dbeaver#34693] [General]
Could anyone say if logic is separated from view in DBeaver? For example, can I create an web application for DB using DBeaver core without eclipse platform?

**Comment by E1izabeth** [comment]
We have a web application. It's called Cloudbeaver. You can find it here https://github.com/dbeaver/cloudbeaver
Also documentation https://dbeaver.com/docs/cloudbeaver/
